app/src/main/java/org/systers/mentorship/view/activities/SendRequestActivity.kt

```diff
@@ -92,14 +101,23 @@ class SendRequestActivity: BaseActivity() {
                 }
             }
 
-            val sendRequestData = SendRequest(
-                    menteeId = menteeId,
-                    mentorId = mentorId,
-                    notes = notes,
-                    endDate = endDate
-            )
 
-            sendRequestViewModel.sendRequest(sendRequestData)
+            if(!TextUtils.isEmpty(notes) && !TextUtils.isEmpty(tvRequestEndDate.text)){
```

Comment:
This if statement can be problematic if the `enddate` is empty and notes is not, this will fall into the else statement and claim notes are empty. Improve this, please. Also be aware of the spaces between if and "(" characters, it is inconsistent with the rest of the code.

you have:
```
if(condition){
     statement
}else{
    other statement
}
```

Put it like this:
```
if (condition) {
     statement
} else {
    other statement
}
```

Reply:
@isabelcosta but endDate can't be empty because in beginning we are setting it to default